script/node-disabled-tests.json

```diff
@@ -26,6 +26,7 @@
   "parallel/test-http2-clean-output",
   "parallel/test-https-agent-session-reuse",
   "parallel/test-https-options-boolean-check",
+  "parallel/test-icu-minimum-version",
```

Comment:
This test was testing like, the wrong thing because it relied on `deps/v8` which Electron doesn't use.  In a follow up we can write an equivalent test in electron.